### DISCUSSION TITLE
Tillæg OK El Flex og Grøn El Flex v/lavt forbrug

### DISCUSSION THREAD

**Discussion by ogregpt** [rndfm/elspotpris#146] [General]
Af aftalen [https://www.ok.dk/betingelser/Betingelser_OK_El_Flex.pdf](url) side 2 fremgår at der er et årligt gebyr på kr. 250 ved årsforbrug under 500 kWh - synes ikke det bliver regnet med ind når jeg prøver?
Gebyret gælder begge aftaler.
Mvh.

**Comment by rndfm** [comment]
Nej den har vi ikke lige fået med :) Gid man havde så lavt et elforbrug.

Som det er nu understøtter siden ikke gebyrer begrænset til et bestemt forbrug. Det kan være det skal understøttes på et tidspunkt.

**Comment by rndfm** [comment]
Man kunne evt. begrænse hele produktet til minimum 500 kWh så det ikke vises under 500 kWh

**Reply by ogregpt** [reply]
Det er forbrug pr. måler, det lave forbrug kunne være i et kolonihavehus, fritidshus eller en separat opsat måler i en garage i et lejlighedskompleks, hvor der er tilsluttet en fryser og/eller nogle maskiner samt lidt belysning...
En begrænsning eller advarsel kunne være fint - jeg ved dog ikke om der er andre abonnementer der har en grænse, eller hvor den i givet fald ligger.
I øvrigt tak for en fin side!
